mockgcp/README.md

```diff
@@ -36,6 +36,9 @@ Broadly the steps are:
 
    * Note: Ensure you pick the same version the controller uses to call the GCP API during GCP resource instantiation.
 
+   * Note: If the resource does not map to an actual GCP resource, you might not be able to find a proto file
```

Comment:
I like that you share the experience and give a concrete example. I think this may not belong to the mockGCP session, because the MockGCP does not care ( or do not know ) the KCC objects.

Reply:
I see your point. Do you have any idea where it would be better suited? 

I wanted to add this because I had some issues initially, trying to figure out the appropriate proto to use for the virtual resource bigquerydatasetaccess. I was following the README guide here under MockGCP when I encountered this, so I added here.